app/controllers/users.js

```diff
@@ -154,6 +157,36 @@ export function create(req, res) {
   }
 }
 
+export const logIn = (req, res) => {
+  const { email, password } = req.body;
+  if (password && email) {
+    if (validator.isEmail(req.body.email)) {
+      User.findOne({
+        email
+      }).then((existingUser) => {
+        if (!existingUser) {
+          return res.status(404).send({ message: 'User not found' });
+        }
+        if (!bcrypt.compareSync(req.body.password, existingUser.hashed_password)) {
+          return res.status(401).send({ message: 'Incorrect Password/Email' });
```

Comment:
how do you think we can make this mesage better for the users `Incorrect Password/Email'`